saskatoon/saskatoon/settings.py

```diff
@@ -122,9 +122,9 @@
     'default': {
         'ENGINE': os.getenv('SASKATOON_DB_ENGINE'),
         'NAME': os.getenv('SASKATOON_DB_NAME'),
-        'USER': os.getenv('SASKATOON_DB_USER'),
-        'PASSWORD': os.getenv('SASKATOON_DB_PASSWORD'),
-        'HOST': os.getenv('SASKATOON_DB_HOST'),
+        # 'USER': os.getenv('SASKATOON_DB_USER'),
+        # 'PASSWORD': os.getenv('SASKATOON_DB_PASSWORD'),
+        # 'HOST': os.getenv('SASKATOON_DB_HOST'),
```

Comment:
Why did you commented those lines? I feel like this should not be part of the changes of this PR.